sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -350,6 +364,22 @@ def service_account(self) -> str:
     def service_account(self, service_account: str):
         self._service_account = service_account
 
+    @classmethod
+    def from_config(cls, config: turing.generated.models.EnsemblerDockerConfig):
```

Comment:
nit: Missing return types for `classmethod`s.

Reply:
Cool, let fix that.